CONTRIBUTING.md

```diff
@@ -1,4 +1,172 @@
 # Contribution Guide
 
-See the [Contribution Guide](https://github.com/pingcap/community/blob/master/CONTRIBUTING.md) in the
-[community](https://github.com/pingcap/community) repo.
+TiDB is a community-driven open source project and we welcome any contributor.  Contributions to the TiDB project are expected to adhere to our [Code of Conduct](https://github.com/pingcap/community/blob/master/CODE_OF_CONDUCT.md).
+
+This document outlines some conventions about development workflow, commit message formatting, contact points and other resources to make it easier to get your contribution accepted. This document is the canonical source of truth for things like supported toolchain versions for building and testing TiDB.
+
+## Hacktoberfest
+
+TiDB is in Hacktoberfest 2020! [Hacktoberfest](https://hacktoberfest.digitalocean.com/) is a month-long celebration & challenge of open source projects organized by DigitalOcean, Intel, and DEV. Whether you are a seasoned contributor or looking for projects to contribute to for the first time, you’re welcome to participate. Open four or more high-quality pull requests between **October 1 and 31** to any public repositories to get a limited edition Hacktoberfest T-shirt!
+
+To participate：
+
+1. Sign up for [Hacktoberfest](https://hacktoberfest.digitalocean.com/login) using your GitHub account.
+2. Pick up an issue from the following issue lists specially tailored for you.
+3. Submit your PRs
+4. Our maintainers review your submitted PRs. Once you successfully merged 4 or more of them, the PRs will be automatically counted, and you will be eligible to claim your SWAG on Hacktoberfest website.
+
+Available issues:
+
+|  Component | Description  | Issue Link   |
+|---|---|---|
+|  TiDB Dashboard | A Web UI for monitoring, diagnosing and managing the TiDB cluster.   | <https://github.com/pingcap-incubator/tidb-dashboard/issues?q=is%3Aopen+is%3Aissue+label%3AHacktoberfest>   |
```

Comment:
I think to move the chapter of `## Hacktoberfest` to the community repo will be better?

Reply:
It will be obscured if we put in the community repo.  This is a temporary update that we want to expose as much as possible. So this is the place.

Reply:
we also have other active community events, should we put them here as well?